install/tools/ipa-replica-conncheck

```diff
@@ -342,7 +398,9 @@ def port_check(host, port_list):
         raise RuntimeError("Port check failed! Inaccessible port(s): %s" \
                 % ", ".join(msg_ports))
 
+
 def main():
+    global RESPONDER
```

Comment:
Nitpick: Feel free to ignore this.

Yuck. It would be better to split the huge `main` into smaller functions and pass `RESPONDER` between those. Anyway, this is mainly relict of the old code so I do not insist on getting rid of it.